doc/source/user/tutorial-svd.rst

```diff
@@ -75,41 +51,38 @@ We can see the image using the `matplotlib.pyplot.imshow` function::
 
 .. note::
 
-   If you are executing the commands above in the IPython shell, it might be
-   necessary to use the command ``plt.show()`` to show the image window. 
-		     
+   If you're using the IPython shell, type ``plt.show()`` if the image fails
+   to appear. You'll need `ctrl-C` to return to IPython.
```

Comment:
Totally true. There's actually a bigger issue here. Procida would say the tutorial dictates the tools, so rather than "If you're using IPython," the tutorial says "Start IPython." 

I have mixed feelings. We'd like people using their favorite tools, but we don't want to have to explain every possible thing that can go wrong on all of them.